### PR TITLE
Error cleanup

### PR DESCRIPTION
**Pull request by raxo2222**

### CODE REVIEW

File: common/production_methods/06_urban_center.txt [hunk 1]
```diff
@@ -2033,7 +2033,7 @@ pm_high_rise_offices = {
 	building_modifiers = {
 		workforce_scaled = {
 			building_input_paper_add = 25
-			building_input_computers_add = 5
+			building_input_electronics_add = 5
```
Comment (DZappala) [inline common/production_methods/06_urban_center.txt:2036]: "Electronics" is an industrial good representing circuit boards. "Computers" is a consumer good and here is in reference to typewriters.

File: common/history/states/00_states.txt
Comment (JackEPepper) [inline common/history/states/00_states.txt:1]: Did you run this change by Max?

**Review by JackEPepper — DISMISSED** [review]
Everything looks good, just want to make sure map changes won't give max a headache

File: common/production_methods/06_urban_center.txt (hunk 1 above)
Comment (raxo2222) [reply common/production_methods/06_urban_center.txt:2036]: I'll remove this input then, as computers is era later compared to this PM

File: common/history/states/00_states.txt
Comment (raxo2222) [reply common/history/states/00_states.txt:1]: He was fine with that - always remember to update two different places when moving province

Comment (Max200012) [reply common/history/states/00_states.txt:1]: everything looks fine map-wise

**Review by JackEPepper — APPROVED** [review]

**Pull request merged**